.gitignore

```diff
@@ -8,6 +8,7 @@
 *.swp
 *~
 .DS_Store
+.vscode
```

Comment:
I've added this to the `.gitignore` so that the VS Code cache files are ignored when developers attempt to commit their changes.